.github/workflows/integration-test.yaml

```diff
@@ -93,7 +91,7 @@ jobs:
         shell: bash
         env:
           ZIP_PASSWORD: ${{ secrets.TEST_USER_PASSWORD }}
-        run: 7z a ./playwright-traces.7z ./bin/playwright-traces/*.* -p"$ZIP_PASSWORD"
+        run: 7z a ./playwright-traces.7z ./frontend/test-results/*/trace.zip -p"$ZIP_PASSWORD"
```

Comment:
I'm pretty sure we want the whole `test-results` folder. The HTML-report is a big win that we didn't get in .NET.

Reply:
Done in commit 0e4d407e.